Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -205,6 +206,38 @@ class KillOldProcesses(shell.Compile):
     command = ["python3", "Tools/CISupport/kill-old-processes", "buildbot"]
 
 
+class PruneCoreSymbolicationdCacheIfTooLarge(buildstep.BuildStep):
+    name = "prune-coresymbolicationd-cache-if-too-large"
+    description = ["pruning coresymbolicationd cache to < 10GB"]
+    descriptionDone = ["pruned coresymbolicationd cache"]
+    flunkOnFailure = True
```

Comment:
Do we really want to fail the build (and have red ews status-bubbles) if this step fails? 
Seems like this and haltOnFailure should be no.

Reply:
Both have been changed to `False`.